src/fillTank.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+"use strict";
 
 /**
```

Comment:
It seems like there is an extra opening comment tag `/**` here, which might be a typo. Ensure that the JSDoc comment block is correctly formatted.